apps/assessment_registry/tests/test_schemas.py

```diff
@@ -301,3 +301,38 @@ def _query_check(**kwargs):
         self.force_login(non_confidential_member_user)
         content = _query_check(okay=False)
         self.assertEqual(content['data']['project']['assessmentRegistries']['totalCount'], 3)
+
+    def test_issue_list_query_filter(self):
+        query = '''
+            query MyQuery (
+                $isParent: Boolean
+                $label: String
+            ) {
+            assessmentRegSummaryIssues(
+                isParent: $isParent
+                label: $label
+                ) {
+                results {
+                    id
+                }
+            }
+        }
+        '''
+
+        member_user = UserFactory.create()
+        self.force_login(member_user)
+
+        parent_issue1, parent_issue2, parent_issue3 = SummaryIssueFactory.create_batch(3, parent=None)
+        child_issue1 = SummaryIssueFactory.create(parent=parent_issue1)
+        child_issue2 = SummaryIssueFactory.create(parent=parent_issue2)
+
+        for filter_data, expected_issues in [
+            ({'isParent': True}, [parent_issue1, parent_issue2, parent_issue3]),
+            ({'isParent': False}, [child_issue1, child_issue2, parent_issue1, parent_issue2, parent_issue3]),
```

Comment:
Related to above comment
```suggestion
            ({}, [child_issue1, child_issue2, parent_issue1, parent_issue2, parent_issue3]),
            ({'isParent': True}, [cparent_issue1, parent_issue2, parent_issue3]),
            ({'isParent': False}, [child_issue1, child_issue2]),
```